demo/admin/src/products/future/generated/CombinationFieldsTestProductsGrid.tsx

```diff
@@ -148,7 +148,7 @@ export function ProductsGrid({ toolbarAction, rowAction }: Props): React.ReactEl
             renderCell: ({ row }) => {
                 const primaryEmptyMessage = <FormattedMessage id="product.staticSelectInStock.primaryText.empty" defaultMessage="No stock info" />;
                 const inStockPrimaryLabels: Record<string, React.ReactNode> = {
-                    true: <FormattedMessage id="product.staticSelectInStock.primaryText.true" defaultMessage="It's in stock :D" />,
+                    true: <FormattedMessage id="product.staticSelectInStock.primaryText.true" defaultMessage={`It's in stock :D`} />,
```

Comment:
Why is this necessary?

Reply:
In case the given string includes double-quotes e.g., here:

https://github.com/vivid-planet/comet/blob/0526d525c62b17eba6fdc0b5ed7614d6c2b59065/demo/admin/src/products/future/generated/CombinationFieldsTestProductsGrid.tsx#L258

Usually prettier replaces the `{``}` with double quotes if they are not necessary, however since this string includes a single quote it is not replaced. 

Should I add a check to only do the `{``}` syntax if the string includes a double quote?